diff --git a/Makefile.am b/Makefile.am
old mode 100644
new mode 100755
index 00b74e5..3d613a7
--- a/Makefile.am
+++ b/Makefile.am
@@ -279,6 +279,7 @@ module_LTLIBRARIES += drm-backend.la
 drm_backend_la_LDFLAGS = -module -avoid-version
 drm_backend_la_LIBADD =				\
 	$(COMPOSITOR_LIBS)			\
+	$(EGL_LIBS)							\
 	$(DRM_COMPOSITOR_LIBS)			\
 	$(INPUT_BACKEND_LIBS)			\
 	libshared.la				\
diff --git a/src/compositor-drm.c b/src/compositor-drm.c
old mode 100644
new mode 100755
index 893877d..0c9402e
--- a/src/compositor-drm.c
+++ b/src/compositor-drm.c
@@ -45,6 +45,7 @@
 
 #include <gbm.h>
 #include <libudev.h>
+#include <GLES2/gl2.h>
 
 #include "compositor.h"
 #include "compositor-drm.h"
@@ -656,7 +657,9 @@ drm_output_repaint(struct weston_output *output_base,
 		}
 		output_base->set_dpms(output_base, WESTON_DPMS_ON);
 	}
-
+	
+	glFinish();
+	
 	if (drmModePageFlip(backend->drm.fd, output->crtc_id,
 			    output->next->fb_id,
 			    DRM_MODE_PAGE_FLIP_EVENT, output) < 0) {
@@ -779,7 +782,9 @@ drm_output_start_repaint_loop(struct weston_output *output_base)
 	 * Use pageflip fallback.
 	 */
 	fb_id = output->current->fb_id;
-
+	
+	glFinish();
+	
 	if (drmModePageFlip(backend->drm.fd, output->crtc_id, fb_id,
 			    DRM_MODE_PAGE_FLIP_EVENT, output) < 0) {
 		weston_log("queueing pageflip failed: %m\n");
